### PR TITLE
Will write (incorrect) TCPU to summary files.

### PR DESCRIPTION
**Pull request by joakim-hove**
Requires: https://github.com/OPM/opm-parser/pull/1067

### CODE REVIEW

**Comment by joakim-hove** [comment]
jenkins build this with downstreams opm-parser=1067 please

File: tests/summary_deck.DATA [hunk 1]
```diff
@@ -55,6 +55,8 @@ FIPNUM
    
 
 SUMMARY
+TCPU
+
 DATE
 PERFORMA
```
Comment (atgeirr) [inline tests/summary_deck.DATA]: Eg trudde PERFORMA impliserte TCPU? Eg kan ha misforstått.

Comment (joakim-hove) [reply tests/summary_deck.DATA]: That might very well be - I will check.

**Comment by joakim-hove** [comment]
jenkins build this with downstreams opm-parser=1067 please

**Comment by joakim-hove** [comment]
jenkins build this with downstreams

**Pull request merged**